patroni/api.py

```diff
@@ -1141,7 +1141,7 @@ def get_postgresql_status(self, retry: bool = False) -> Dict[str, Any]:
             * ``server_version``: Postgres version without periods, e.g. ``150002`` for Postgres ``15.2``;
             * ``xlog``: dictionary. Its structure depends on ``role``:
                 * If ``master``:
-                    * ``location``: ``pg_current_wal_lsn()``
+                    * ``location``: ``pg_current_wal_flush_lsn()``
```

Comment:
I have a feeling, we have to reveal these details in the user documentation for API as well, along with the versions' peculiarities
`Current location of the Postgres transaction log` doesn't look like enough info.
Wdyt?

Reply:
9.6 and even 10 are EOL, I don't think it is worth time spend on it.

Reply:
ok, regardless the older versions, there are still quite some versions of what can be the "Current location" here